Modules/Utils/Sources/PocketCastsUtils/Sorting/PodcastSorter.swift

```diff
@@ -8,8 +8,12 @@ public enum PodcastSorter {
      - Returns true when title1 is alphabetically before title2, false otherwise
      */
     public static func titleSort(title1: String, title2: String) -> Bool {
-        let convertedTitle1 = title1.trimmingThePrefix()
-        let convertedTitle2 = title2.trimmingThePrefix()
+
+        var convertedTitle1 = title1.trimmingThePrefix()
+        var convertedTitle2 = title2.trimmingThePrefix()
+
+        convertedTitle1 = convertToPinyinIfNeeded(convertedTitle1)
```

Comment:
By moving the func to an extension you can simple do `convertedTitle1.convertToPinyinIfNeeded()`

Or even:

```
var convertedTitle1 = title1.trimmingThePrefix().convertToPinyinIfNeeded()
         var convertedTitle2 = title2.trimmingThePrefix().convertToPinyinIfNeeded()
```

Reply:
Like it. Much more streamlined. Done!